cmake/scripts/SquirrelExport.cmake

```diff
@@ -267,7 +267,7 @@ foreach(LINE IN LISTS SOURCE_LINES)
     endif()
 
     # Maybe the end of the class, if so we can start with the Squirrel export pretty soon
-    if("${LINE}" MATCHES "};")
+    if("${LINE}" MATCHES "^(	*)};")
```

Comment:
literal tab in a regex feels troublesome. `[\t]*` ? Or perhaps some better brace matching ;)

Reply:
Done in #14329.